src/node/chainstate.cpp

```diff
@@ -99,7 +99,7 @@ static ChainstateLoadResult CompleteChainstateInitialization(
     // by a call to `chainman.MaybeRebalanceCaches()`. We just need to make sure
     // that the sum of the two caches (40%) does not exceed the allowable amount
     // during this temporary initialization state.
-    double init_cache_fraction = 0.2;
```

Comment:
The comment above needs to be edited. At least "Conservative value which is arbitrarily chosen" should make clear that it's referring to 0.2. And the reference to `MaybeRebalanceCache` should probably also be changed.

Reply:
Seemed clear to me that the comment refers to the smaller factor, but reading it again, I think you are right. Will re-word.